### DISCUSSION TITLE
Two connected Search results webparts: Hide results if no items are selected

### DISCUSSION THREAD

**Discussion by spiderb4by** [microsoft-search/pnp-modern-search#3622] [Q&A]
I might need your help with something.

I created a cross-search between a list of items and associated documents by following the instructions here: https://microsoft-search.github.io/pnp-modern-search/scenarios/Connect-to-a-search-results-webpart/

With the configuration of the two Search Results webparts, I observe the expected behavior: when an item is selected, the associated documents are displayed. Which is fine!

The problem I'm having is that when no items are selected in webpart 1, webpart 2 displays by default a list of ALL documents in my library. Is there a way to hide the results in the second webpart, if nothing is selected?

Here is a view of my Search Results:
<img width="360" alt="image" src="https://github.com/microsoft-search/pnp-modern-search/assets/160788261/c6bb49a9-b1af-440c-ad0b-bb39d23df959">

**Comment by dontthroworanges** [comment]
Bump! I have the exact same question.

**Comment by kasperbolarsen** [comment]
I guess you are using the "Process selected values as filters" option?
![image](https://github.com/user-attachments/assets/6ec5123e-5a18-46f4-b2b6-2b86ce4de37d)


In this blog post I am using the "as tokens" option and it sounds like that is what you are looking for:
https://www.m365thinking.com/post/displaying-attached-files-belonging-to-a-list-item-in-search
